lib/pymedphys/tests/dicom/test_connect.py

```diff
@@ -187,3 +198,41 @@ def test_dicom_listener_send_conflicting_file(listener, test_dataset):
 
     # Clean up after ourselves
     shutil.rmtree(storage_path)
+
+
+@pytest.mark.pydicom
+def test_dicom_listener_cli(test_dataset):
+
+    scp_ae_title = "PYMEDPHYSTEST"
+
+    with process(
+        f"poetry run pymedphys dicom listen {TEST_PORT} -d {HERE} -a {scp_ae_title}",
```

Comment:
I suspect calling `poetry run` here will break Darcy's pydicom master push:

https://github.com/pydicom/pydicom/blob/e58a922c132246a6a9b236eae1fe0015ff9d8879/.github/workflows/master-push.yml#L148-L149

When his tests are run they are run from the production pymedphys install. As such, no poetry available. Does this process appropriately scope if it's simply called as `pymedphys dicom listen`?

Reply:
It runs for me locally, however I'm not across poetry enough to understand exactly where we need to be using that... :/